### ISSUE TITLE
How to scroll panel into view when it is shown on click?

### ISSUE THREAD

**Issue by gnoyixiang** [react-component/collapse#96]
Hello! 
I would like the panel to be scrolled into view (if it is not in view) when the user clicks it. 
I cannot get the DOM element by applying ref to the Collapse or Panel component. 
Kindly advise. Thanks!

**Comment by marko1943** [comment]
any luck with this? 😄

**Comment by gnoyixiang** [comment]
Nope, havent tried out anything yet. But I think the way to do it is apply ref to the children node of the Panel. That should work and should be the way it is.
